alfasim_sdk/cli.py

```diff
@@ -58,7 +59,27 @@ def template(dst, caption, plugin_id, author_name, author_email):
     dst = Path(dst)
     hook_specs_file_path = _get_hook_specs_file_path()
     hm = HookManGenerator(hook_spec_file_path=hook_specs_file_path)
-    hm.generate_plugin_template(caption, plugin_id, author_email, author_name, dst)
+    alfasim_sdk_include = ["<alfasim_sdk_api/alfasim_sdk.h>"]
+    default_impls_for_hooks = [
+        "HOOK_INITIALIZE(ctx){" "    return 0;" "}",
```

Comment:
Why are you using 3 consecutive strings?
For all that python care those are a single continuous string.

Reply:
Last commit fix this issue =)